plugin/storage/grpc/shared/grpc_client.go

```diff
@@ -117,7 +82,7 @@ func (c *GRPCClient) ArchiveSpanWriter() spanstore.Writer {
 
 // GetTrace takes a traceID and returns a Trace associated with that traceID
 func (c *GRPCClient) GetTrace(ctx context.Context, traceID model.TraceID) (*model.Trace, error) {
-	stream, err := c.readerClient.GetTrace(upgradeContext(ctx), &storage_v1.GetTraceRequest{
```

Comment:
where is the code that attaches interceptors to the client that would do these upgrades instead?

Reply:
Interceptors are being attached to shared client in `/plugin/storage/grpc.go` https://github.com/jaegertracing/jaeger/blob/6a520a570dd192935aaa01da29a8c9c732f01277/plugin/storage/grpc/factory.go#L162